src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -1,5 +1,10 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
 
+### New Rules
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+CA2250 | Usage | Info | UseCancellationTokenThrowIfCancellationRequested, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2250)
```

Comment:
The suggested category in the runtime issue was "Style". @mavasani @sharwell Should this be an IDExxxx instead?

Reply:
I didn't find a "Style" category in the `DiagnosticCategory` class. Should I add it?

Reply:
> Should I add it?

@NewellClark Probably not. Better to wait for @mavasani or @sharwell feedback on this.

Reply:
I would suggest using "Usage" category. Style is reserved for whitespace formatting and usage of new language features.